package.json

```diff
@@ -1,48 +0,0 @@
-{
-  "name": "@caerostris/parallel-transform-stream",
```

Comment:
Does it make sense to publish with a prefix? There’s no module with that name and it’s developed @ubilabs.